components/sensor/pzem004t.rst

```diff
@@ -67,3 +83,4 @@ See Also
 - :doc:`pzemdc`
 - :apiref:`pzem004t/pzem004t.h`
 - :ghedit:`Edit`
+- :link:https://www.home-assistant.io/integrations/utility_meter/
```

Comment:
```suggestion
- `utility_meter <https://www.home-assistant.io/integrations/utility_meter/>`__
```

See contributing guide, this is the way to create links.
Also the link would fit better in the `Example config for energy` section.